### ISSUE TITLE
NullReferenceException thrown while typing package ID

### ISSUE THREAD

**Issue by AArnott** [dotnet/ProjFileTools#35]
### Installed product versions
- Visual Studio: 2017 Enterprise (27513.1.d15.7)
- This extension: 1.1.0

After typing `<PackageReference Include="`, VS put up an error dialog from the extension.

![image](https://user-images.githubusercontent.com/3548/37489226-3431fbf6-2854-11e8-8c9d-729955bb0915.png)

```
---------------------------
Assertion Failed: Abort=Quit, Retry=Debug, Ignore=Continue
---------------------------
System.AggregateException: One or more errors occurred. ---> System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.<ExecuteAutocompleteServiceQueryAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.<GetPackageNamesAsync>d__9.MoveNext()
   --- End of inner exception stack trace ---
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.TransformToPackageInfo(Task`1 arg)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exce......
<truncated>
---------------------------
Abort   Retry   Ignore   
---------------------------
```

The activitylog.xml file reveals more of the callstack:

```
System.AggregateException: One or more errors occurred. ---&gt; System.AggregateException: One or more errors occurred. ---&gt; System.NullReferenceException: Object reference not set to an instance of an object.
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;ExecuteAutocompleteServiceQueryAsync&gt;d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;GetPackageNamesAsync&gt;d__9.MoveNext()
   --- End of inner exception stack trace ---
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.TransformToPackageInfo(Task`1 arg)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at ProjectFileTools.NuGetSearch.Search.PackageFeedSearchJob`1.HandleCompletion(Task`1 task)
   at ProjectFileTools.NuGetSearch.Search.PackageFeedSearchJob`1..ctor(IReadOnlyList`1 searchTasks, CancellationTokenSource cancellationTokenSource)
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.SearchPackageNamesInternal(String prefix, String tfm, IPackageQueryConfiguration config)
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.&lt;&gt;c__DisplayClass7_0.&lt;SearchPackageNames&gt;b__1(PackageNameQuery x)
   at System.Collections.Concurrent.ConcurrentDictionary`2.AddOrUpdate(TKey key, Func`2 addValueFactory, Func`3 updateValueFactory)
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.SearchPackageNames(String prefix, String tfm)
   at ProjectFileTools.Completion.PackageCompletionSource.AugmentCompletionSession(ICompletionSession session, IList`1 completionSets)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.CompletionSession.Start()
   at ProjectFileTools.Completion.CompletionController.StartSession()
   at ProjectFileTools.Completion.CompletionController.Exec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.CommandChainNode.InnerExec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.CommandChainNode.Exec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at ProjectFileTools.GotoDefinitionController.Exec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.CommandChainNode.InnerExec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.CommandChainNode.InnerExec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.SimpleTextViewWindow.Exec(Guid&amp; pguidCmdGroup, UInt32 nCmdID, UInt32 nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
   at Microsoft.VisualStudio.Editor.Implementation.VsKeyboardFilter.SendCommand(Guid cmdGroup, UInt32 cmdID, Object inParam)
   at Microsoft.VisualStudio.Editor.Implementation.VsKeyboardFilter.SendCommand(VSStd2KCmdID cmdID, Object inParam)
   at Microsoft.VisualStudio.Editor.Implementation.VsKeyboardFilter.TextInput(TextCompositionEventArgs args)
   at Microsoft.VisualStudio.Text.Editor.Implementation.KeyProcessorDispatcher.&lt;&gt;c.&lt;DispatchTextInputEvents&gt;b__6_0(KeyProcessor p, TextCompositionEventArgs args)
   at Microsoft.VisualStudio.Text.Editor.Implementation.KeyProcessorDispatcher.&lt;&gt;c__DisplayClass10_1`1.&lt;Dispatch&gt;b__0()
   at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.CallExtensionPoint(Object errorSource, Action call)
---&gt; (Inner Exception #0) System.AggregateException: One or more errors occurred. ---&gt; System.NullReferenceException: Object reference not set to an instance of an object.
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;ExecuteAutocompleteServiceQueryAsync&gt;d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;GetPackageNamesAsync&gt;d__9.MoveNext()
   --- End of inner exception stack trace ---
   at ProjectFileTools.NuGetSearch.Search.PackageSearchManager.TransformToPackageInfo(Task`1 arg)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
---&gt; (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object.
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;ExecuteAutocompleteServiceQueryAsync&gt;d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ProjectFileTools.NuGetSearch.Feeds.Web.NuGetV3ServiceFeed.&lt;GetPackageNamesAsync&gt;d__9.MoveNext()&lt;---
&lt;---
```

**Comment by mlorbetske** [comment]
This should be fixed with #36

**Comment by mlorbetske** [comment]
The fix has been merged
